### ISSUE TITLE
Add `description_md` to spell out acronyms, where needed

### ISSUE THREAD

**Issue by mccalluc** [hubmapconsortium/ingest-validation-tools#883]
Please do all assays in a single PR.

Here are the `assay_type`s that might benefit:

```
            'scRNA-Seq(10xGenomics)',
            'AF',
            'bulkATACseq',
            'Cell DIVE',
            'CODEX',
            'CE-MS',
            'DESI',
            'LC-MS (metabolomics)',
            'LC-MS/MS (label-free proteomics)',
            'MxIF',
            'MALDI-IMS',
            'MS (shotgun lipidomics)',
            'NanoDESI',
            'NanoPOTS',
            'PAS microscopy',
            'scATACseq',
            'sciATACseq',
            'sciRNAseq',
            'seqFISH',
            'SIMS-IMS',
            'SNARE-seq2',
            'snATACseq',
            'snRNA',
            'SPLiT-Seq',
            'TMT (proteomics)',
            'WGS',
            'SNARE2-RNAseq',
```

**Comment by cebriggs7135** [comment]
@chuck what is the deliverable here? Have you created a blank page for me to populate or ??

**Comment by cebriggs7135** [comment]
@chuck How do I provide these descriptions to you?

**Comment by mccalluc** [comment]
You will make a branch and edit each of the assay YAMLs in turn, and then make a PR for that branch.

**Comment by mccalluc** [comment]
Fixed by #890.
